plugins/nf-sqldb/build.gradle

```diff
@@ -43,6 +91,8 @@ dependencies {
     api 'org.postgresql:postgresql:42.2.23'
     api 'org.xerial:sqlite-jdbc:3.36.0.3'
     api 'org.duckdb:duckdb_jdbc:0.3.0'
+    implementation name: 'AthenaJDBC42_2.0.25.1001'
+    api "org.slf4j:log4j-over-slf4j:1.7.32"
```

Comment:
Instead, this should be `compileOnly` because at runtime it's inherited from the core nextflow module